fixity/utils.py

```diff
@@ -36,5 +36,11 @@ def utcnow():
     return datetime.now(timezone.utc)
 
 
+def format_timestamp(t):
+    return t.strftime("%Y-%m-%d %H:%M:%S %Z")
+
+
 def pyprint(message, **kwargs):
+    if kwargs.get("timestamps"):
+        message = f"{[format_timestamp(utcnow())]} {message}"
```

Comment:
If your intention is to recreate the [brackets in ansible-archivematica-src](https://github.com/artefactual-labs/ansible-archivematica-src/blob/d4c964c8f037fcb18e84af7dfcc804a23dab5075/templates/bin/fixity-cron#L21) I think you need to move the square brackets out of the curly ones. This is creating a list with a single string in it and you can see its side effect in the expected output of your test:

```python
    assert (
        captured.err.strip()
        == f"['2018-01-01 03:00:00 UTC'] Fixity scan succeeded for AIP: {aip_id}"
    )
```

Those single quotes around the timestamp should not be there.